tooling/postcss-tailwind-css-modules/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://json.schemastore.org/package",
+  "name": "@accelint/postcss-tailwind-css-modules",
+  "title": "Postcss tailwind css modules plugin",
+  "description": "Postcss plugin for resolving some problems that occur when using tailwind with css modules",
+  "author": "https://hypergiant.com",
+  "owner": "default/pathfinder",
+  "keywords": [
+    "dev-tk",
+    "tooling"
+  ],
+  "version": "0.0.1",
+  "private": false,
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gohypergiant/standard-toolkit"
+  },
+  "type": "module",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    }
+  },
+  "scripts": {
+    "build": "tsup",
+    "constellation-tracker": "node ../constellation-tracker/dist/main.js",
+    "lint": "pnpm biome lint"
+  },
+  "devDependencies": {
+    "@accelint/typescript-config": "workspace:*",
+    "@types/node": "^24.9.0",
+    "postcss": "^8.5.6",
```

Comment:
Would these need to be technically expressed as peerDeps as well? I think we may be fine but want to sanity check